asset/js/base.js

```diff
@@ -29,6 +29,25 @@
     // Tooltip
     $('[data-tooltip]').tooltip();
 
+    // Component selector
+    var componentSelector = document.querySelector('.component-selector__control');
+    if (componentSelector) {
+        componentSelector.addEventListener('change', function (event) {
+            var value = event.target.value;
+            if (value.length === 0) {
+                return;
+            }
+
+            // Create URL
+            var url              = new URL(
```

Comment:
Extra whitespace can be omitted here. :smiley:

Reply:
Automated IDE function. Compare with line 47. 😄

Reply:
I feel a zend-javascript-coding-style coming up :)

Reply:
Felt strange, as the definitions are not on adjacent lines.

I'm not going to enforce a CS for JS assets; this just stood out as odd! :smile: